src/main/java/org/fundacionjala/coding/ovidio/FizzBuzz.java

```diff
@@ -0,0 +1,64 @@
+package org.fundacionjala.coding.ovidio;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by OvidioMiranda on 8/25/2017.
+ */
+public class FizzBuzz {
+    private  List<String> number;
+    private static final String FIZZ_BUZZ = "FizzBuzz";
+    private static final String FIZZ = "Fizz";
+    private static final String BUZZ = "Buzz";
```

Comment:
these constants should be public or package access in order to use them in the test class.
Because currently you are repeating the code. :P